_i18n/de.yml

```
@@ -0,0 +1,90 @@
+meta:
+  description: Entwickle 2D- und 3D-Spiele, plattformübergreifenden Projekte oder sogar XR-Ideen!
+header:
+  features: Features
+  showcase: Showcase
+  blog: Blog
+  community: Community
+  assets: Assets
+  download: Download
+  docs: Docs
+  contribute: Mitwirken
+  donate: Spenden
+  language: Sprache
+footer:
+  download: Download
+  documentation: Dokumentation
+  features: Features
+  web_editor: Web Editor
+  release_archive: Release Archiv
+  source_code: Quellcode
+  project: Projekt
+  blog: Blog
+  code_of_conduct: Verhaltenskodex
+  governance: Verwaltung
+  teams: Teams
+  priorities: Prioritäten
+  communities: Communities
+  resources: Ressourcen
+  asset_library: Asset-Bibliothek
+  press_kit: Presse-Kit
```

```suggestion
  press_kit: Presse
```
I think German websites tend to just say this, because "Pressemappe" feels strange while technically correct.